src/app/main.go

```diff
@@ -34,32 +87,26 @@ func badgeHandler(w http.ResponseWriter, r *http.Request) {
 	}
 
 	const (
-		height    = 20
-		padding   = 7
-		charWidth = 7
-		radius    = 3
+		height  = 20
+		padding = 7
+		radius  = 3
 	)
 
-	kindWidth := charWidth * len(kind)
-	labelWidth := charWidth * len(label)
+	kindWidth := measureStringWidth(kind)
+	labelWidth := measureStringWidth(label)
 	width := 4*padding + kindWidth + labelWidth
 
 	s := svg.New(w)
 	s.Start(width, height)
-	s.Style("text/css", `
-		text {
-			font-family: DejaVu Sans,Verdana,Geneva,sans-serif;
-			font-size: 11px;
-			fill: white;
-		}`)
+	s.Style("text/css", textStyle)
 	s.LinearGradient("gradient", 0, 0, 0, 255, []svg.Offcolor{
 		{Offset: 0, Color: "#000", Opacity: 0},
 		{Offset: 200, Color: "#000", Opacity: 0.25},
 		{Offset: 225, Color: "#000", Opacity: 1.0},
 	})
 	s.Roundrect(0, 0, width, height, radius, radius, "fill:#666")
 	s.Roundrect(padding+kindWidth, 0, 3*padding+labelWidth, height, radius, radius, "fill:#"+color)
-	s.Rect(padding+kindWidth, 0, 10, height, "fill:#"+color)
+	s.Rect(padding+kindWidth, 0, 10, height, "fill:#666")
```

Comment:
why?

Reply:
This fixes #1 (it's in the commit message).  Otherwise, the badge looks like ![b0rked](https://tia.mat.br/temp/status.svg) instead of the nicer ![fixed](https://tia.mat.br/temp/status-fixed.svg).

Reply:
oh! sounds good